Nef_3/include/CGAL/Nef_3/SNC_external_structure.h

```diff
@@ -255,34 +255,50 @@ class SNC_external_structure_base : public SNC_decorator<SNC_structure_>
   typedef CGAL::Unique_hash_map<SFace_const_handle,bool> SFace_visited_hash;
   typedef CGAL::Unique_hash_map<SFace_const_handle,bool> Shell_closed_hash;
 
+  typedef std::vector<SFace_handle> SFace_list;
+  typedef std::vector<Halffacet_handle> Halffacet_list;
```

Comment:
I would not call a vector a list.    Maybe `SFace_handles` just does it?

Reply:
Yes, and I also realise these two typedefs can be moved to the Shell_entry struct. Should I also make the new implementation details private?